homeassistant/components/samsungtv/media_player.py

```diff
@@ -62,19 +66,21 @@
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the Samsung TV from a config entry."""
-    ip_address = config_entry.data[CONF_IP_ADDRESS]
+    host = config_entry.data[CONF_HOST]
```

Comment:
I don't see where we ever disconnect the websocket when the entity is removed from hass or the stop event is fired

Reply:
Added